internal/cli/atlas/privateendpoints/aws/interfaces/delete.go

```diff
@@ -38,18 +38,18 @@ func (opts *DeleteOpts) initStore() error {
 }
 
 func (opts *DeleteOpts) Run() error {
-	return opts.Delete(opts.store.DeleteInterfaceEndpoint, opts.ConfigProjectID(), provider, opts.privateEndpointID)
+	return opts.Delete(opts.store.DeleteInterfaceEndpoint, opts.ConfigProjectID(), provider, opts.privateEndpointServiceID)
 }
 
-// mongocli atlas privateEndpoint(s) aws interface(s) delete <atlasPrivateEndpointId> [--privateEndpointId privateEndpointID][--projectId projectId]
+// mongocli atlas privateEndpoint(s) aws interface(s) delete <atlasPrivateEndpointId> [--privateEndpointServiceID privateEndpointServiceID][--projectId projectId]
 func DeleteBuilder() *cobra.Command {
 	opts := &DeleteOpts{
 		DeleteOpts: cli.NewDeleteOpts("Interface endpoint '%s' deleted\n", "Interface endpoint not deleted"),
 	}
 	cmd := &cobra.Command{
 		Use:     "delete <atlasPrivateEndpointId>",
```

Comment:
delete and describe seem to be using different names, can we please make sure both and for both cloud providers use the same name for the param, I'd probably go for 
```suggestion
		Use:     "delete <endpointId>",
```